sqlmesh/core/config/root.py

```diff
@@ -171,16 +176,32 @@ def _normalize_and_validate_fields(cls, values: t.Dict[str, t.Any]) -> t.Dict[st
                     f"The `{plan_deprecated}` config is deprecated. Please use the `plan.{plan_deprecated}` config instead."
                 )
 
+        if values.get("physical_schema_override") and values.get("physical_schema_mapping"):
```

Comment:
Should we check for key existence instead? Eg. this check will still pass if one of the dicts is empty.

Reply:
Unless keys are always present in this validator, of course

Reply:
Yes, good point. I poked around in the debugger and the only keys that are present are the ones that get passed in when the `Config` object is created, so checking for key existence makes sense